lib/onc_certification_g10_test_kit/bulk_data_group_export.rb

```diff
@@ -161,7 +161,7 @@ class BulkDataGroupExport < Inferno::TestGroup
         start = Time.now
 
         loop do
-          get(polling_url, headers: { authorization: "Bearer #{bearer_token}" })
+          get(polling_url, headers: { authorization: "Bearer #{bearer_token}", accept: 'application/fhir+json' })
```

Comment:
It should be 'application/json'

"When requesting status, the client SHOULD use an Accept header indicating a content type of application/json."